src/OrbitGl/TrackManager.cpp

```diff
@@ -140,7 +147,7 @@ void TrackManager::SortTracks() {
   sorted_tracks_.clear();
 
   // Scheduler track.
-  if (!scheduler_track_->IsEmpty()) {
+  if (scheduler_track_ != nullptr && !scheduler_track_->IsEmpty()) {
```

Comment:
That's exactly the point here: I'm no longer assuming the scheduler track MUST exist. It can be created by the application, but the `TrackManager` does no longer rely on this.

Reply:
ok